Sources/FoundationExtensions.swift

```diff
@@ -134,3 +134,19 @@ extension DispatchTimeInterval {
 		return result
 	}
 }
+
+extension Collection where Element == Bool {
+	/// Returns logical or over all elements or `false` if `self` is empty.
+	///
+	/// - returns: Logical or over all elements or `false` if `self` is empty
+	internal func or() -> Bool {
+		return reduce(false) { $0 || $1 }
+	}
+	
+	/// Returns logical or over all elements or `false` if `self` is empty.
+	///
+	/// - returns: Logical or over all elements or `false` if `self` is empty
+	internal func and() -> Bool {
+		return reduce(true) { $0 && $1 }
+	}
+}
```

Comment:
I'd prefer not to add these extensions. They pollute the namespace and aren't used widely enough to justify that—nor do they add that much over `.reduce(true, &&)`.